scripts/run-test262.ts

```diff
@@ -40,12 +40,17 @@ type Test = {
 };
 
 function shouldSkip(test: Test) {
+  if (test.file.endsWith('.md')) {
+    return true;
+  }
+
   const features = test.attrs.features ?? [];
   return features.some((feature) => unsupportedFeatures.has(feature));
 }
 
 async function runTest() {
   await run(parse, {
+    testsDirectory: path.dirname(require.resolve('test262/package.json')),
```

Comment:
On my machine, the default value uses a different copy from 'test262-parser-runner', adding this will enforce to use the correct version in package.json.